scripts/bundleSizeConfig.js

```diff
@@ -2,5 +2,5 @@ module.exports = {
   // Size limit for all bundles used by each service (K)
   // Keep these +/- 5K and update frequently!
   MIN_SIZE: 702,
```

Comment:
I just checked locally on your PR for the smallest bundled service atm, which is currently 724.
![image](https://user-images.githubusercontent.com/30599794/76513341-132a7e00-644e-11ea-87d1-75767cc21752.png)

so we should also update the min size here to be -5 of 724